M2/Macaulay2/packages/LLLBases.m2

```diff
@@ -1248,6 +1248,7 @@ TEST ///
 
 TEST 
 /// -- DON'T TEST YET??
+    -- no-check-flag (encountered an unknown key or option: Engine)
```

Comment:
@mikestillman - This test appears to never have been run since `TEST` and the test string were on separate lines.  After changing `TEST` to a keyword, the test does get run, but it fails because `LLL` doesn't have an `Engine` option.

I'm proposing that we continue skipping it for now (which `no-check-flag` takes care of), but I figured I'd tag you in case you'd like to take a look.

Reply:
Was there an engine implementation that got removed or never got added? Seems strange.